src/Surfnet/StepupRa/RaBundle/Resources/public/less/style.less

```diff
@@ -180,6 +180,25 @@ select.form-control {
     }
 }
 
+form[name="ra_management_create_ra"] {
```

Comment:
Maybe it's better to add a class to the button group and determine the styling based on that 'modifier' class.